server/src-lib/Hasura/RQL/Types/Common.hs

```diff
@@ -89,16 +92,31 @@ import           Hasura.SQL.Backend
 
 
 type family ScalarType (b :: Backend) where
-  ScalarType 'Postgres = PGScalarType
+  ScalarType 'Postgres = PG.PGScalarType
 
 type family ColumnType (b :: Backend) where
-  ColumnType 'Postgres = PGType
+  ColumnType 'Postgres = PG.PGType
 
 type family Column (b :: Backend) where
-  Column 'Postgres = PGCol
+  Column 'Postgres = PG.PGCol
 
 type family SQLExp (b :: Backend) where
-  SQLExp 'Postgres = S.SQLExp
+  SQLExp 'Postgres = PG.SQLExp
+
+class
+  ( Show (TableName b)
+  , Eq (TableName b)
+  , Lift (TableName b)
+  , NFData (TableName b)
+  , Cacheable (TableName b)
+  , Hashable (TableName b)
+  , Data (TableName b)
+  , Typeable b
+  ) => Representation (b :: Backend) where
```

Comment:
We rename `Backend` to `BackendType` and `Representation` to `Backend`?

Reply:
Ah, yeah, that works! I've pushed a comment, I'll do the renaming.

Reply:
This is an elegant solution.

Reply:
Done!